front/pages/api/w/[wId]/data_sources/[name]/documents/[documentId]/index.ts

```diff
@@ -38,9 +35,19 @@ async function handler(
   res: NextApiResponse<WithAPIErrorResponse<GetDocumentResponseBody>>,
   auth: Authenticator
 ): Promise<void> {
-  const plan = auth.getNonNullablePlan();
+  const { documentId, name } = req.query;
 
-  const dataSource = await getDataSource(auth, req.query.name as string);
+  if (typeof name !== "string" || typeof documentId !== "string") {
```

Comment:
Shouldn't we use io-ts here? Wondering why we sometimes use it and sometimes not

Reply:
We're using iots for body parsing, here it's path parameters - but we could definitely use iots even if it's only simple strings.